easybuild/easyconfigs/d/DL_POLY_Classic/DL_POLY_Classic-1.9-foss-2015b-PLUMED-2.2.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'MakeCp'
+
+name = 'DL_POLY_Classic'
+version = '1.9'
+
+homepage = 'http://ccpforge.cse.rl.ac.uk/gf/project/dl_poly_classic/'
+description = """DL_POLY Classic is a freely available molecular dynamics program developed
+ from the DL_POLY_2 package. This version does not install the java gui."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = ['dl_class_%(version)s.tar.gz']
+source_urls = ['http://ccpforge.cse.rl.ac.uk/gf/download/frsrelease/255/2627/']
+
+plumedversion = '2.2.0'
+versionsuffix = '-PLUMED-%s' % plumedversion
+
+dependencies = [('PLUMED', plumedversion)]
+
+# this patch needs to be regenerated for another version of PLUMED
+patches = [('PLUMED-%s.patch' % plumedversion, 1)]
```

Comment:
procedure is (partially) explained at https://groups.google.com/forum/#!topic/plumed-users/cWaIDU5F6Bw

@wpoely86: can you give a detailed overview of what else needs to be done to automate these patches?